packages/office-ui-fabric-react/src/components/Fabric/Fabric.styles.ts

```diff
@@ -4,18 +4,18 @@ import { IFabricStyleProps, IFabricStyles } from './Fabric.types';
 const inheritFont = { fontFamily: 'inherit' };
 
 const GlobalClassNames = {
-  root: 'ms-Fabric'
+  root: 'ms-Fabric',
+  bodyThemed: 'ms-Fabric-body-themed'
```

Comment:
Not super familiar with class name conventions but I think this would typically be `ms-Fabric-bodyThemed`?

Reply:
Yeah I see that's how isFocusVisible is too - camelcasing